shared/utils/cli/utils.go

```diff
@@ -0,0 +1,33 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// Print a TX's details to the console.
+func PrintTransactionHash(hash common.Hash) {
+
+    hashString := hash.String()
+
+    fmt.Printf("Transaction has been submitted with hash %s.\n", hashString)
+    fmt.Printf("You may follow its progress by visiting:\n")
+    fmt.Printf("https://goerli.etherscan.io/tx/%s\n\n", hashString)
```

Comment:
shouldn't hard-code the url, either leave it out or put into config

Reply:
Agreed, I put it into the config: https://github.com/rocket-pool/smartnode-install/commit/ea2f26be32e6abc8e5cab34be3ac4b72397f5e69